mythril/laser/ethereum/instructions.py

```diff
@@ -811,7 +809,7 @@ def mstore_(self, global_state):
         try:
             # Attempt to concretize value
             _bytes = util.concrete_int_to_bytes(value)
-            state.memory[mstart : mstart + len(_bytes)] = _bytes
+            state.memory[mstart : mstart + 32] = _bytes
```

Comment:
What happens when _bytes does not have 32 bytes?

Reply:
fortunately that doesn't happen as util.concrete_int_to_bytes gives a 32 length word.So this line was obsolete.Maybe adding an assert there might be a good thing to do

Reply:
added #684